pl_engine/pl_server/src/main/java/com/cubrid/jsp/ExecuteThread.java

```diff
@@ -315,7 +316,10 @@ private void processStoredProcedure() throws Exception {
         ctx.checkTranId(tid);
 
         StoredProcedure procedure = makeStoredProcedure(unpacker);
+
+        sendAuthCommand(0, procedure.getAuthUser());
```

Comment:
Replacing 0 and 1 with mnemonics would be better for readability.
Are they 'push user' and 'pop user'?

Reply:
Good point. I'm going to add enum values for them.

Reply:
I just add pushUser() and popUser() functions for now. When I merge with other sub-issues related to DEFINER and CURRENT_USER, I'm going to add the enum values and refactor this routine for readability in the parent issue.